src/components/AppShell.astro

```diff
@@ -0,0 +1,14 @@
+---
+import Header from './Header.vue';
+import DesktopNavigation from './DesktopNavigation.astro';
+const { path, navigation } = Astro.props;
+
+const navPath = path.endsWith('/') ? path.slice(0, -1) : path;
+---
+
+  <div class="flex h-screen w-screen">
```

Comment:
@alexBinns are you missing a closing div? 🤨

Reply:
You are right...fixed